profiles/pminio/compose.yaml

```diff
@@ -0,0 +1,20 @@
+version: "3.7"
+
+services:
+  pminio:
+    image: minio/minio
+    environment:
+      - "MINIO_ACCESS_KEY={S3_ACCESS_KEY}"
+      - "MINIO_SECRET_KEY={S3_SECRET_KEY}"
+    ports:
+      - "{MINIO_PORT}:9000"
+      - "{MINIO_CONSOLE_PORT}:9090"
+    volumes:
+      - "oci_pminio:/data"
+    depends_on:
+      - pulp
```

Comment:
Should not pulp depend on this service instead?

Reply:
Any ideas on how to achieve that, doesn't seem obvious to me. Technically it is not needed to be up first for Pulp to start.